selfdrive/test/process_replay/process_replay.py

```diff
@@ -336,12 +338,34 @@ def ublox_rcv_callback(msg):
   ),
 ]
 
+def setup_prefix():
+  os.environ['OPENPILOT_PREFIX'] = str(uuid.uuid4())
+  msg_path = '/dev/shm/' + os.environ['OPENPILOT_PREFIX']
+  try:
+    os.mkdir(msg_path)
+  except FileExistsError:
+    pass
+  except PermissionError:
+    print(f"WARNING: failed to make {msg_path}")
```

Comment:
we don't want to handle this here. it's only in manager since manager shouldn't really crash
```suggestion
```